.env.example

```diff
@@ -0,0 +1,9 @@
+#
+# Copy this to `.env` and customize to your needs.
+#
+
+# packages/core
+DB_HOST=localhost
+DB_USER=
+DB_PASS=
+DB_NAME=sup_bot
```

Comment:
@orta Figured we could have a single .env file for the monorepo?

Reply:
We should, yep!